twisted/conch/ssh/keys.py

```diff
@@ -782,18 +801,24 @@ def public(self):
         return Key(self._keyObject.public_key())
 
 
-    def fingerprint(self):
+    def fingerprint(self, format=FingerprintFormats.MD5_HEX):
         """
-        Get the user presentation of the fingerprint of this L{Key}.  As
-        described by U{RFC 4716 section
-        4<http://tools.ietf.org/html/rfc4716#section-4>}::
+        The fingerprint of a public key consists of the output of the
+        message-digest algorithm in the specified format.
+        Supported formats inclide C{md5-hex} and C{sha256-base64}
```

Comment:
These need to be updated to be `L{}` to the correct things. Also, `inclide` should be `include`.